### DISCUSSION TITLE
Error after upgrading macOS to Ventura 13.1

### DISCUSSION THREAD

**Discussion by yusenguo** [firedrakeproject/firedrake#2735] [Firedrake support] [answered]
Hi, I have just upgraded my mac 2019 to macOS Ventura 13.1 from Catalina. The code used to work now runs into an error while importing firedrake:

xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
Traceback (most recent call last):
  File "/Users/yusenguo/firedrake/ex_heat_equation/Helmholtz.py", line 1, in <module>
    from firedrake import *
  File "/Users/yusenguo/firedrake/src/firedrake/firedrake/__init__.py", line 34, in <module>
    _blas_lib_path = petsc.get_blas_library()
  File "/Users/yusenguo/firedrake/src/firedrake/firedrake/petsc.py", line 125, in get_blas_library
    petsc_c_library = [f for f in petsc_py_dependencies if "libpetsc" in f][0]
IndexError: list index out of range

Would you please help me fix this issue?

**Comment by dham** [comment, marked as answer]
Upgrading a major OS version will break your Xcode command line tools, and probably a bunch of libraries that Firedrake depends on. You should run `brew doctor` and do all the things it tells you to do, and then do a fresh install of Firedrake.

**Reply by yusenguo** [reply]
Will do. Thanks a lot.
